src/script/repositories/conversation/ConversationRepository.ts

```diff
@@ -593,7 +593,7 @@ export class ConversationRepository {
   /**
    * Get a conversation from the backend.
    */
-  public async fetchConversationById({id: conversationId, domain}: QualifiedId): Promise<Conversation> {
+  private async fetchConversationById({id: conversationId, domain}: QualifiedId): Promise<Conversation> {
```

Comment:
this was made public here https://github.com/wireapp/wire-webapp/pull/19236

This method is not suitable for refreshing the cells state, as the database manipulation is creating rendering issues